### DISCUSSION TITLE
Pause charging button

### DISCUSSION THREAD

**Discussion by Jseeca** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1297] [Ideas]
Hey, just a suggestion but in addition to a top up button there should be a pause charge button. 

Why is this? There are times when I'm on the go and I'm using a powerbank to charge my mac. But power banks tend to only be around 80% efficient and when fast charging (50-100w) the efficiency can drop as low as 60%! That kind of defeats the purpose of the powerbank if the useable capacity is only half of what is rated. 

The solution I came up of is to use Aldente is pause the charging at whatever percentage it was at, so the mac only feeds off the power bank which usually amounts to 5-15w giving a better efficiency rating. This can allow me to have an extra 20-30% more power off gird than if I just were to charge it as it was. 

At the moment I'm just manually changing the set percentage to my current percentage but a feature like that would be good!

**Comment by MatthiasKerbl** [comment]
Hi @Jseeca,

Thank you for your suggestion. Another solution for your scenario would be to use the right-click feature. Go to Settings/Features/Menubar Right Click and choose "Charging Start/Stop". Afterward, when you right-click on the menubar icon, the charge limit will change between the current battery percentage and 100%.

Best,
Matthias
